src/layer-manager/layer-manager.test.js

```diff
@@ -0,0 +1,22 @@
+import * as layerManager from './index'
+import { validate as mapboxStyleValidator } from '@mapbox/mapbox-gl-style-spec'
+
+const glyphPath =
+  'https://raw.githubusercontent.com/GlobalFishingWatch/map-gl-glyphs/master/_output/{fontstack}/{range}.pbf?raw=true'
+
+test('returns the valid glyphs path', () => {
+  const objectToMatch = {
+    glyphs: glyphPath,
+  }
+  expect(layerManager.getMapStyle(glyphPath)).toMatchObject(objectToMatch)
+  // expect(layerManager.getMapStyle(glyphPath)).toMatchSnapshot()
+})
+
+test('check valid style.json format', () => {
+  const style = layerManager.getMapStyle(glyphPath)
+  const errors = mapboxStyleValidator(style)
```

Comment:
I'm wondering if it could be useful to have that running at runtime too (only in dev obviously)